pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -1995,12 +2175,27 @@ func (tc *testContext) verify(t *testing.T, expected result, initialObjects []me
 	}
 }
 
+// sortClaim sorts given claim's Requests in spec, and Results in status
+func sortClaim(claim *resourceapi.ResourceClaim) {
```

Comment:
Why is this necessary? Isn't the order deterministic?

Reply:
not really, each run may produce different order.

Reply:
I removed all calls to `sortClaim` and ran the unit tests 100 times without any failures.

Are you seeing something different?

Or did you mean "may" as in "is allowed to, even if it does not currently"? I disagree, it should not be allowed to produce random results.

Reply:
the pod allocation results are not deterministic, i think the randomness is from the allocator. For the example run given above, the request container-0-request-0 is allocated with one device, the one device could be instance-3, or could be another instance, say instance-1